applications/plugins/SofaCarving/examples/CarvingTool.scn

```diff
@@ -5,7 +5,7 @@
         <RequiredPlugin name="Sofa.Component.Collision.Detection.Intersection"/> <!-- Needed to use components [MinProximityIntersection] -->  
         <RequiredPlugin name="Sofa.Component.Collision.Geometry"/> <!-- Needed to use components [PointCollisionModel, TriangleCollisionModel] -->  
         <RequiredPlugin name="Sofa.Component.Collision.Response.Contact"/> <!-- Needed to use components [CollisionResponse] -->  
-        <RequiredPlugin name="Sofa.Component.Constraint.Projective"/> <!-- Needed to use components [FixedConstraint, LinearMovementConstraint] -->  
+        <RequiredPlugin name="Sofa.Component.Constraint.Projective"/> <!-- Needed to use components [FixedConstraint, LinearMovementProjectiveConstraint] -->
```

Comment:
```suggestion
        <RequiredPlugin name="Sofa.Component.Constraint.Projective"/> <!-- Needed to use components [LinearMovementProjectiveConstraint] -->
```

There is also an occurence of `FixedConstraint` in this file. Could you check again in the examples of all the code base that you renamed the components?